.github/workflows/ci.yaml

```diff
@@ -66,7 +65,13 @@ jobs:
             miniforge-version: latest
             activate-environment: pypsa-eur
             use-mamba: true
-            
+
+      - name: Cache data bundle
+        uses: actions/cache@v3
+        with:
+          path: data/bundle
```

Comment:
Maybe also `cutouts`?

Reply:
Maybe also everything else in `data`, not only `data/bundle`? 

I mean, this stuff is not supposed to change, right?

Reply:
Also relevant, if you plan on using it to cache the natura 2000 GPKG download ;-)

Reply:
Done, but will now reset cache every week to check that retrieve rules are still functioning.